crates/jstz_api/src/http/response.rs

```diff
@@ -268,6 +268,21 @@ impl ResponseBuilder {
         //
         // FIXME: jstz only supports status, body and headers here.
         // FIXME: http::Response won't support 0 as a status code
+        let status = 400;
```

Comment:
This technically isn't spec compliant, `Response.error()` should return a "network error" (status 0, empty body, empty headers). Perhaps this is what we want, since the uses of `Response::internal_server_error` are actually for network related issues (i.e. transfer failed)